src/centralized_telescope/algorithm.rs

```diff
@@ -52,6 +52,24 @@ pub fn verify(setup: &Setup, proof: &Proof) -> bool {
     proof_hash(setup, &round)
 }
 
+/// Alba's proving algorithm used for benchmarking, returning a boolean
+/// according to whether a proof has been found as well as the number of steps
+/// and reties done.
+pub fn bench(setup: &Setup, prover_set: &[Element]) -> (u64, u64, bool) {
```

Comment:
I don't see a way to make this function non-public unfortunately, but we can at least do this.
```
pub mod internal {
  pub fn bench() {}
}
```

Reply:
What do you mean by removed? it's still here.

Reply:
I need to rebase the code, but we shouldn't need bench any longer as it was needed only for repetitions, which we decided to remove.

Reply:
Don't we need still need it for the step benchmark?

Reply:
Yes I have just noticed... The only thing I could use without another function is the retry  step * max dfs bound but we would have the same variance problem as for the nb of repetitions.

I'll add an internal module for the time being and try again with features afterwards

Reply:
After rebasing I cannot do this directly anymore because of the wrapper.

I can either:
- put an internal module in algorithm but I need to 1) change the algorithm mod to public and 2) get a function to extract the Setup from the wrapper,
- or add a bench function in the wrapper, but this cannot be in a module (not allowed in impl block).

Reply:
You can change `mod algorithm;` back to `pub mod algorithm;` in `src/centralized_telescope/mod.rs` and do what you were doing before. `prove()` and `verify()` in `algorithm.rs` are `pub(super)` anyway.

Reply:
To get `Setup`, you can just call `make_setup()` directly.